test/610_proxy_wait_for_weave_test.sh

```diff
@@ -12,4 +12,10 @@ fi
 
 assert_raises "proxy docker_on $HOST1 run -e 'WEAVE_CIDR=10.2.1.1/24' $BASE_IMAGE $CHECK_ETHWE_UP"
 
+# Check committed containers only have one weavewait prepended
+LAST_CONTAINER=$(proxy docker_on $HOST1 ps -a -q -n=1)
+COMMITTED_IMAGE=$(proxy docker_on $HOST1 commit $LAST_CONTAINER)
+assert_raises "proxy docker_on $HOST1 run --name c2 $COMMITTED_IMAGE"
+assert "proxy docker_on $HOST1 inspect --format='{{.Config.Entrypoint}}' c2" "{[/w/w]}"
```

Comment:
or perhaps check that the entrypoint of this container is the same as its parent?